filament/src/main/java/net/fabricmc/filament/task/base/FilamentTask.java

```diff
@@ -0,0 +1,20 @@
+package net.fabricmc.filament.task.base;
+
+import javax.inject.Inject;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.Internal;
+
+import net.fabricmc.filament.FilamentExtension;
+
+public abstract class FilamentTask extends DefaultTask {
+	@Inject
+	public FilamentTask() {
+		setGroup("filament");
+	}
+
+	@Internal
+	public FilamentExtension getExtension() {
```

Comment:
```suggestion
	protected FilamentExtension getExtension() {
```
(Assuming it compiles ofc)